src/core/Akka/Actor/TypedActor.cs

```diff
@@ -34,7 +35,10 @@ public abstract class TypedActor : ActorBase
         /// <returns>TBD</returns>
         protected sealed override bool Receive(object message)
         {
-            MethodInfo method = GetType().GetMethod("Handle", new[] {message.GetType()});
+            Type[] msgTypeArray = new[] {message.GetType()};
+            Type actorType = GetType();
+            MethodInfo method = actorType.GetMethod("Handle", msgTypeArray) ??
+                                actorType.GetMethod($"Akka.Actor.IHandle<{msgTypeArray[0].FullName}>.Handle", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
Didnt know that. I could replace this with only reflection API calls but as always with reflection it could be sloppy